src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -176,32 +178,42 @@ export const Sidebar: React.FC<Props> = ({
     }
   }, [redeemCode, campaignData, refetchAll])
 
+  const isNorwegianBundle = isBundle(offerData) && isNorwegian(offerData)
   const discountText = getDiscountText(textKeys)(
     redeemedCampaigns,
     offerData.cost.monthlyGross.currency,
   )
+  const clearDiscountCode = () => {
+    removeDiscountCode()
+      .then(() => {
+        const cookieStorage = new CookieStorage()
+        cookieStorage.setItem('_hvcode', '', {
+          path: '/',
+        })
+      })
+      .then(() => refetchAll())
+  }
 
-  const isNorwegianBundle = isBundle(offerData) && isNorwegian(offerData)
+  const hasRedeemedCampaigns = redeemedCampaigns.length > 0
 
   return (
     <>
       <ReactVisibilitySensor partialVisibility onChange={setIsSidebarVisible}>
         {() => (
           <Wrapper>
             <Container>
-              <HeaderTop>
-                <HedvigSymbol />
+              {discountText && (
                 <DiscountInfo>
                   {isNorwegianBundle && (
-                    <Badge>{textKeys.SIDEBAR_NO_BUNDLE_DISCOUNT_TEXT()}</Badge>
+                    <DiscountBadge>
+                      {textKeys.SIDEBAR_NO_BUNDLE_DISCOUNT_TEXT()}
+                    </DiscountBadge>
                   )}
-                  {discountText && <Badge>{discountText}</Badge>}
+                  <DiscountBadge>{discountText}</DiscountBadge>
                 </DiscountInfo>
-              </HeaderTop>
-
+              )}
               <Header>
-                <Title>{getInsuranceTitle(offerData, textKeys)}</Title>
-
+                <Title>Hedvig</Title>
```

Comment:
Should this really be hard coded? I realise there no translations needed but still feels a bit odd. Or have you discussed that?

Reply:
Since it is the brand name, I thought of it as a sort of "logo". But as you said, this does not really belong in translations. What do you suggest?